tests/cli/grass/test_grass.py

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+
+import json
+import logging
+import os
+import time
+import unittest
+import uuid
+
+import yaml
+
+from maro.cli.utils.params import GlobalParams, GlobalPaths
+from maro.cli.utils.subprocess import SubProcess
+
+
+@unittest.skipUnless(os.environ.get("test_with_cli", False), "Require cli prerequisites.")
+class TestGrass(unittest.TestCase):
+    """Tests for Grass Mode.
+
+    Tests should be executed in specific order,

```

Comment:
```python
import unittest
unittest.TestLoader.sortTestMethodsUsing = None
```
This one may also work for this case.

ref: https://stackoverflow.com/questions/4095319/unittest-tests-order/22317851#22317851

Reply:
apparently, a better way is to define a sorting function to determine the strategy we want to order, and that will incur some redundant workload. So we are not going to change it at this time.